routers/api/v1/api.go

```diff
@@ -454,7 +454,7 @@ func mustAllowPulls(ctx *context.APIContext) {
 }
 
 func mustEnableIssuesOrPulls(ctx *context.APIContext) {
-	if !ctx.Repo.CanRead(models.UnitTypeIssues) &&
+	if (!setting.Repository.EnableIssues || !ctx.Repo.CanRead(models.UnitTypeIssues)) &&
```

Comment:
So even the repository created some issues, this will still hide them when `setting.Repository.EnableIssues` changed to true?